scipion/install/plugin_funcs.py

```diff
@@ -2,29 +2,26 @@
 import os
 import re
 import sys
-from importlib import import_module
 import json
 import pkg_resources
 from pkg_resources import parse_version
 
+from funcs import Environment
 from pyworkflow.plugin import Domain
 from pyworkflow.utils.path import cleanPath
 from pyworkflow import LAST_VERSION, CORE_VERSION, OLD_VERSIONS, Config
-from pyworkflow.install import Environment
-
+from importlib import reload
 REPOSITORY_URL = Config.SCIPION_PLUGIN_JSON
 
 if REPOSITORY_URL is None:
     REPOSITORY_URL = Config.SCIPION_PLUGIN_REPO_URL
 
 PIP_BASE_URL = 'https://pypi.python.org/pypi'
-PIP_CMD = '{0} {1}/pip install %(installSrc)s'.format(
-    Environment.getBin('python'),
-    Environment.getPythonPackagesFolder())
+PIP_CMD = '{0} -m pip install %(installSrc)s'.format(
```

Comment:
I think we have a Python defined in Environment and also in pyworkflow.Config, we should review this.

Reply:
We haven't reviewed this part.....I think with latest changes....all can be handled by each setup.py and remove the addPipModule...??